server/express/test.rest

```diff
@@ -0,0 +1,8 @@
+POST http://localhost:3333/login HTTP/1.1
+Content-type: application/json
+
+{
+    "token": "eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiIiLCJjbGFpbURhdGEiOnsiYmxvY2tOdW1iZXIiOiIxMzYxMTY1MiJ9LCJzdWIiOiIiLCJpYXQiOjE2MzE4NjEwMDUzOTZ9.MHgwOWVkNWUyZWY3MmRiNGI1MWQzZjQ1YjJjZDVmMDdhOTVmNzEyZDM3NzQ3NWQ0ZDFkMDg0ODg2ZDc4OTdmYmU4MzE0OTdjZjg2YmY5ZjM5N2Q0YzU5NDkwZWE3ZjI3MWE1ZjgyYWZhODdmNzkzZTFmZGY3OTZhZjMzYWNjNDAxMjFj"
+}
+
+# # Authorization: token eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiIiLCJjbGFpbURhdGEiOnsiYmxvY2tOdW1iZXIiOiIxMzYxMTY1MiJ9LCJzdWIiOiIiLCJpYXQiOjE2MzE4NjEwMDUzOTZ9.MHgwOWVkNWUyZWY3MmRiNGI1MWQzZjQ1YjJjZDVmMDdhOTVmNzEyZDM3NzQ3NWQ0ZDFkMDg0ODg2ZDc4OTdmYmU4MzE0OTdjZjg2YmY5ZjM5N2Q0YzU5NDkwZWE3ZjI3MWE1ZjgyYWZhODdmNzkzZTFmZGY3OTZhZjMzYWNjNDAxMjFj
```

Comment:
What is this file for? Is it a test that is runnable?

Reply:
Oh I should remove it, thanks. 
It is a file I use to quickly test my APIs using [REST Client](https://marketplace.visualstudio.com/items?itemName=humao.rest-client) plugin from vscode